### PR TITLE
Search,edit and delete of GraphQL API in API Catalog test(ONEPLAT-1724)

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- Yes
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: e2e/integration/apicatalogtests.js [hunk 1]
```diff
@@ -67,10 +65,126 @@ context( 'API Catalog Tests', () => {
...
+        cy.get('.graphql-voyager').should('be.visible')
+    } )
+    it( 'Going back', () => {
+        cy.go('back')
+    } )
+    it( 'Click and Open GraphQL Playground', () => {
+        cy.contains( 'GraphQL Playground' ).click( { force: true } )
+        cy.wait( 10000 );
+        cy.get( '.sc-fONwsr.jNppkf.playground' ).should( 'be.visible' )
+        cy.go('back')
```
Comment (riginoommen) [inline e2e/integration/apicatalogtests.js:165]: this test may be outdated testing playground is not needed

**Comment by riginoommen** [comment]
SquAH THE COMMITS commit history is polluted

**Comment by riginoommen** [comment]
@mudit94 Hi any update on this PR

**Comment by mudit94** [comment]
stale closing it

**Pull request closed without merging**